src/addons/sourcemod/scripting/zr/hgversion.h.inc

```diff
@@ -1,7 +1,9 @@
 #define ZR_VER_PRODUCT_NAME	    "Zombie:Reloaded"
-#define ZR_VER_COPYRIGHT        "Copyright (C) 2009-2023  Greyscale, Richard Helgeby, maxime1907, BotoX, zaCade, .Rushaway, Franug, Anubis"
-#define ZR_VER_VERSION          VERSION
+#define ZR_VER_COPYRIGHT        "Copyright (C) 2009-2025  Greyscale, Richard Helgeby, maxime1907, BotoX, zaCade, .Rushaway, Franug, Anubis"
 #define ZR_VER_BRANCH           "master"
-#define ZR_VER_REVISION         ""
+#define ZR_VER_MAJOR            "3"
+#define ZR_VER_MINOR            "11"
+#define ZR_VER_REVISION         "0"
```

Comment:
ZR_VER_PATCH

Reply:
I removed the revision since it was never used since the creation of the plugin.